README.md

```diff
@@ -1,3 +1,53 @@
 # ⚠️ Work in progress
 
 ## Do not use in production!
+
+
+## Releasing
+
+We're using Sonatype OSSRH to host our open source project binaries.
+Docs: https://central.sonatype.org/publish/publish-guide/
+
+#### 1. Get access to be able to deploy
+
+1. First create an account to access JIRA and later the repository manager https://issues.sonatype.org/secure/Signup!default.jspa
+2. Create a ticket similar to https://issues.sonatype.org/browse/OSSRH-59076 & get one of the people who already have access to comment on the request with approval (you can ask in #team-platform channel).
```

Comment:
really? wow

Reply:
yeeeep

Reply:
just fyi: also gives you access to Android releases or anything else we might have in the future under `com.posthog`